api/helpers/gamestate/moves/points/validate.js

```diff
@@ -0,0 +1,52 @@
+// api/helpers/gameState/moves/points/validate.js
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Parse points request',
+
+  description: 'Returns needed values to make points requests',
+
+  inputs: {
+    requestedMove: {
+      type: 'ref',
+      description: 'Object containing data needed for current move',
+      required: true,
+    },
+    /**
+     * @param {Object} requestedMove - Object describing the request to play points
+     * @param {1 | 0} requestedMove.playedBy - Which player is playing
+     * @param { Card } requestedMove.cardPlayed - Card Played for points
+     * @param { MoveType.POINTS } requestedMove.moveType - Specifies that this a Points move
+     */
+    currentState: {
+      type: 'ref',
+      descriptions: 'Object containing the current game state',
+      required: true,
+    },
+  },
+  sync: true, // synchronous helper
+  fn: ({ requestedMove, gameState }, exits) => {
```

Comment:
We should add a check that the requesting user is in the game. Should that go here, in parserequest, or in post move?

Reply:
I think it should go in the move endpoint so that we only need it in one place, since it's a requirement of every move